Wrong shot number:
[6, 9, 8]

Correct shot number:
[10]

Raw json:

{'hasError': True, 'errParameter': ['dfs.datanode.volumes.replica-add.threadpool.size'], 'reason': ["The property 'dfs.datanode.volumes.replica-add.threadpool.size' has the value '0.1' which is not within the accepted value {0,1}."]}

{'hasError': True, 'errParameter': ['dfs.datanode.volumes.replica-add.threadpool.size'], 'reason': ["The property 'dfs.datanode.volumes.replica-add.threadpool.size' has the value '0.1' which is not within the accepted value range {0, 1}."]}

{'hasError': True, 'errParameter': ['dfs.datanode.volumes.replica-add.threadpool.size'], 'reason': ["The property 'dfs.datanode.volumes.replica-add.threadpool.size' has the value '0.1' which is not within the accepted value {0,1}."]}

Final result:

There are 1 misconfiguration parameters in the input: dfs.datanode.volumes.replica-add.threadpool.size
[Ciri] Reason for dfs.datanode.volumes.replica-add.threadpool.size: The property 'dfs.datanode.volumes.replica-add.threadpool.size' has the value '0.1' which is not within the accepted value {0,1}.
